packages/gatsby/src/bootstrap/load-themes/index.ts

```diff
@@ -106,16 +141,29 @@ const processTheme = (
     )
   } else {
     // if a theme doesn't define additional themes, return the original theme
-    return [{ themeName, themeConfig, themeSpec, themeDir, parentDir: rootDir }]
+    return Promise.resolve([
```

Comment:
Slightly modifies the return type to make it consistent with the other `if` branch. End result stays the same.